.github/workflows/test-ui-e2e-only-desktop.yml

```diff
@@ -164,6 +191,50 @@ jobs:
         shell: bash
         run: echo "::notice title=Allure report URL::${{ steps.allure-server.outputs.report-url }}"
 
+      - name: Download Xray Reports
+        if: ${{ !cancelled() && inputs.export_to_xray }}
+        uses: actions/download-artifact@v4.1.1
+        with:
+          path: "apps/ledger-live-desktop/artifacts/xray"
+          pattern: xray-reports-*
+          merge-multiple: false
+
+      - name: Extract and Aggregate Xray reports
+        if: ${{ !cancelled() && inputs.export_to_xray }}
+        run: 
+          ./aggregate-xray-reports.sh
+      
+      - name: Upload aggregated xray reports
+        if: ${{ !cancelled() && inputs.export_to_xray }}
+        uses: actions/upload-artifact@v4.3.0
+        with:
+          name: aggregated-xray-reports
+          path: "apps/ledger-live-desktop/artifacts/xray/aggregated-xray-reports.json"
+
+      - name: Authenticate to Xray
+        if: ${{ !cancelled() && inputs.export_to_xray }}
+        id: authenticate
+        run: |
+          response=$(curl -H "Content-Type: application/json" -X POST --data '{"client_id": "${{ secrets.XRAY_CLIENT_ID }}", "client_secret": "${{ secrets.XRAY_CLIENT_SECRET }}"}' https://xray.cloud.getxray.app/api/v2/authenticate)
```

Comment:
I believe XRAY_API_URL and JIRA_URL could be declared as env vars for better readability